src/locales/ko/dialogue.ts

```diff
@@ -385,182 +385,182 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      1: "내 손으로 널 파묻어주지. 영광으로 알도록!"
     },
     "victory": {
-      1: "Ugh! You are... quite capable...\nI fell behind, but only by an inch..."
+      1: "윽! 꽤… 제법이군…\n내가 뒤쳐졌지만, 겨우 한 끝이다…"
     },
     "defeat": {
-      1: "Team Magma will prevail!"
+      1: "마그마단이 앞선다!"
     }
   },
   "magma_boss_maxie_2": {
     "encounter": {
-      1: "You are the final obstacle remaining between me and my goals.\nBrace yourself for my ultimate attack! Fuhahaha!"
+      1: "나와 내 목표 사이에 남은 마지막 장애물이로군.\n최후의 공격에 대비하도록! 후하하하!"
     },
     "victory": {
-      1: "This... This is not.. Ngh..."
+      1: "이런… 이럴 수는… 없어… "
     },
     "defeat": {
-      1: "And now... I will transform this planet to a land ideal for humanity."
+      1: "이제부터… 인류의 이상을 위해 대지를 더 늘려가겠다."
     }
   },
   "aqua_boss_archie_1": {
     "encounter": {
-      1: "I'm leader of Team Aqua, so I'm afraid it's the rope's end for you."
+      1: "내가 아쿠아단의 리더다, 유감스럽게도 넌 이게 끝이겠군."
     },
     "victory": {
-      1: "Let's meet again somewhere. I'll be sure to remember that face."
+      1: "다시 어딘가에서 만나도록하지. 그 얼굴 기억해둘테니까."
     },
     "defeat": {
-      1: "Brilliant! My team won't hold back now!"
+      1: "좋군! 이제 아쿠아단을 막아설 것은 없다!"
     }
   },
   "aqua_boss_archie_2": {
     "encounter": {
-      1: "I've been waiting so long for this day to come.\nThis is the true power of my team!"
+      1: "이 날이 오는 것을 너무 오래 기다렸군.\n이것이 우리의 진정한 힘이다!"
     },
     "victory": {
-      1: "Like I figured..."
+      1: "내가 생각했던 것들이…… "
     },
     "defeat": {
-      1: "I'll return everything in this world to its original, pure state!!"
+      1: "이 세상을 원래의 순수한 상태로 되돌리겠다, 그 모든 것을!!"
     }
   },
   "galactic_boss_cyrus_1": {
     "encounter": {
-      1: "You were compelled to come here by such vacuous sentimentality\nI will make you regret paying heed to your heart!"
+      1: "넌 공허한 감정에 이끌려 이곳에 오게된 것이다.\n마음에 귀를 기울인 것을 후회하게 해주지!"
     },
     "victory": {
-      1: "Interesting. And quite curious."
+      1: "흥미롭군. 게다가 꽤 궁금해졌어."
     },
     "defeat": {
-      1: "I will create my new world..."
+      1: "새로운 세상을 만들 것이다…"
     }
   },
   "galactic_boss_cyrus_2": {
     "encounter": {
-      1: "So we meet again. It seems our fates have become intertwined.\nBut here and now, I will finally break that bond!"
+      1: "이런 우리가 다시 만나게 됐군. 우리의 운명이 얽히게 된 것 같군.\n 하지만 지금 여기서, 나는 마침내 그 끈을 끊을 것이다!"
     },
     "victory": {
-      1: "How? How? HOW?!"
+      1: "어떻게? 어째서? 어떻게?!"
     },
     "defeat": {
-      1: "Farewell."
+      1: "작별이로군."
     }
   },
   "plasma_boss_ghetsis_1": {
     "encounter": {
-      1: "I won't allow anyone to stop me! No matter who does what!"
+      1: "누구도 저를 막을 수 없습니다! 무슨 짓을 하더라도!"
     },
     "victory": {
-      1: "How can this be? I'm the creator of Team Plasma! I'm perfect!"
+      1: "…어떻게 된 거지? 이 몸은 플라스마단을 만들어낸 완벽한 남자라고! "
     },
     "defeat": {
-      1: "I am the perfect ruler of a perfect new world! Mwa ha ha!"
+      1: "세계를 변하게 할 완벽한 지배자란 말이다! 마 하 하!"
```

Comment:
```suggestion
      1: "세계를 바꿀 완전한 지배자가 아니었단 말인가!? …흐하하!"
```
https://youtu.be/jx649erM1k4?si=gu7bxoALb5RkEEaB&t=763
https://youtu.be/jx649erM1k4?si=JHr9zF_TYuM6CNPm&t=774
블화 원본 대사를 찾아왔습니다

Reply:
오 감사합니다 ㅋㅋㅋ 마 하 하 웃는 게 로컬라이징이 어떻게 됐나...?
바하하하 (1세대 고스트 타입 사천왕, 국화) 후하하하 (1세대 독타입 관장/2세대 사천왕, 독수) 같은경우는 특이한 웃음소리를 잘 살려주더라구요 .. 국제적으로 200x년시절부터... 
그래서 결정도 못내리고 + 게치스가 웃는장면이 있었던가 고민하고 있었어요 ㅋㅋㅋㅋ 감사합니다~~!